e3sm_diags/logger.py

```diff
@@ -12,23 +10,6 @@
 LOG_FILEMODE = "w"
 LOG_LEVEL = logging.INFO
 
-
-# Setup the root logger with a default log file.
-# `force` is set to `True` to automatically remove root handlers whenever
-# `basicConfig` called. This is required for cases where multiple e3sm_diags
-# runs are executed. Otherwise, the logger objects attempt to share the same
-# root file reference (which gets deleted between runs), resulting in
-# `FileNotFoundError: [Errno 2] No such file or directory: 'e3sm_diags_run.log'`.
-# More info here: https://stackoverflow.com/a/49202811
-logging.basicConfig(
-    format=LOG_FORMAT,
-    filename=LOG_FILENAME,
-    filemode=LOG_FILEMODE,
-    level=LOG_LEVEL,
-    force=True,
-)
-logging.captureWarnings(True)
```

Comment:
The root logger is not longer instantiated at the module level in `logger.py`. This will add another layer of protection against the possibility of duplicate logger messages from appearing (which can happen if `logging.basicConfig` is called multiple times.